modules/vector-search/pages/vector-search.adoc

```diff
@@ -0,0 +1,49 @@
+= Use Vector Search for AI Applications
+:page-topic-type: concept
+:description: Use Couchbase Server's Vector Search features to add fast and accurate semantic search to your applications.
+
+[abstract]
+{description}
+
+== About Vector Search
+
+Vector Search builds on Couchbase Server's xref:search.adoc[Search Service] to provide vector index support.
+You can use these new Vector Search indexes for Retrieval Augmented Generation (RAG) with an existing Large Language Model (LLM). 
+
+Using Vector Search, Couchbase Server, an embedding model, and your chosen LLM, you can develop AI applications while giving context and up-to-date information from your own data.
+
+You can develop applications that include: 
+
+* *Similarity search:* Search for documents, products, images, and more that are similar to a given query.
```

Comment:
Should this be Similarity Search?

Reply:
It's not a proper noun, so that's why I have it lowercased :)